docs/source/tutorials/mantid_basic_course/fitting_data/01_fitting_models_to_data.rst

```diff
@@ -48,9 +48,13 @@ Simple fitting
 4. Choose ExpDecay and click OK
 |ChooseExpDecay.png|
 
-5. Notice how the ExpDecay function has appeared in the Functions list on the Fit Property Browser, and there are preset Settings below.
+5. Notice how the ExpDecay function has appeared in the Functions list on the Fit Property Browser, and there are pre-set Settings below.
 For now, just click on the drop-down menu "Fit" and run a normal Fit.
-|RunFitOption.png|
+
+.. figure:: /images/RunFitOption.png
```

Comment:
why is only this image input as a `.. figure`?

Reply:
Before my updates, these were mostly inline images. Here it is a figure to aide formatting.